app/rest/registry/catalog.js

```diff
@@ -0,0 +1,16 @@
+angular.module('portainer.rest')
+.factory('RegistryCatalog', ['$resource', 'API_ENDPOINT_REGISTRIES', function RegistryCatalogFactory($resource, API_ENDPOINT_REGISTRIES) {
+  'use strict';
+  return $resource(API_ENDPOINT_REGISTRIES + '/:id/v2/_catalog?n=:limit&last=:last', {}, {
+    get: {
+      method: 'GET',
+      params: { id: '@id', limit: '@limit', last: '@last' },
+      transformResponse: function(data, headers){
+        var response = {}
+        response.data = JSON.parse(data);
```

Comment:
Why is the data parsed in JSON here while it's not done in the `RegistryBlobs.head` method?

Reply:
Hum, cause we don't use data in the `RegistryBlobs.head` call 😄 
I'll add that soon